Userland/Libraries/LibGUI/Calendar.h

```diff
@@ -8,13 +8,15 @@
 #pragma once
 
 #include <AK/String.h>
+#include <LibConfig/Listener.h>
 #include <LibCore/DateTime.h>
 #include <LibGUI/Frame.h>
 #include <LibGUI/Widget.h>
 
 namespace GUI {
 
-class Calendar final : public GUI::Frame {
+class Calendar final : public GUI::Frame
```

Comment:
Put the first subclass onto a new line as well (`clang-format` misbehaving here)
```c++
class Calendar final
    : public GUI::Frame
    , public Config::Listener {
```